src/api/digitalMedia/ScheduleDigitalMediaMAS.ts

```diff
@@ -0,0 +1,66 @@
+/* Import Dependencies */
+import axios from 'axios';
+import KeycloakService from 'app/Keycloak';
+
+/* Import Exceptions */
+import { PostException } from 'app/Exceptions';
+
+/* Import Types */
+import { JSONResultArray, Dict } from 'app/Types';
+
+
+/**
+ * Function for posting a scheduling job for scheduling MASs for the digital specimen
+ * @param handle The identifier of the digital media item to schedule MASs for
+ * @param masList A list of MASs to be scheduled
+ * @returns Array of scheduled MASs
+ */
+const ScheduleDigitalMediaMAS = async ({ handle, masList }: { handle: string, masList: { masId: string }[] }) => {
+    let digitalMediaMAS: Dict = {};
+
+    if (handle) {
+        const token = KeycloakService.GetToken();
+
+        const masRecord: {
+            data: {
+                type: 'MasRequest',
+                attributes: {
```

Comment:
i think this is meant to be: 
```
attributes: {
            mass: [
                {
                    masId: string
                }
            ]
}
```

Reply:
This is the typing and when you want to indicate a type is an array in TypeScript, you put the block brackets after the value, for example, an array containing strings would be: string[], or an array of objects holding a name: {name: hello}[]